src/utils/linkResolver.js

```diff
@@ -1,4 +1,4 @@
-import { defaultLanguage } from '../../prismic-config'
+const { defaultLanguage } = require('../../prismic-configuration')
```

Comment:
Is there a reason you're not using ES5 imports for this?

Reply:
At some point in time gatsby didn't like people using ES5 in modules that would be used in gatsby-node.

I can check and find out if this is still the case :)

Reply:
I've found a thread on github where they talk about this:https://github.com/gatsbyjs/gatsby/issues/7810
And I tried to [work with a fix they provide](https://github.com/gatsbyjs/gatsby/issues/7810#issuecomment-601748815), didn't work, kept reading and realised it stopped working for everyone else a few months ago.
So maybe we can leave it like this?

Reply:
Yeah, that's fine.